data/categories.txt

```diff
@@ -4428,6 +4429,7 @@ pl:3^Wyjazd|wyjście
 pt:4^Saída|3junção
 pt-BR:2^Saída|3junção
 ro:^Ieșire
+es:^Salida|3intersección
```

Comment:
Note: we will start suggesting "intersección" after 3 letters, i.e. after you type "int".
Is this really important enough to provoke a suggestion?

Reply:
I have taken the inspiration from "en" and "pt"